resources/core_endpoints.go

```diff
@@ -0,0 +1,394 @@
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ObjectMeta.SelfLink"),
+			},
+			{
+				Name:        "uid",
+				Description: "UID is the unique in time and space value for this object",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ObjectMeta.UID"),
+			},
+			{
+				Name:        "resource_version",
+				Description: "An opaque value that represents the internal version of this object that can be used by clients to determine when objects have changed",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ObjectMeta.ResourceVersion"),
+			},
+			{
+				Name:        "generation",
+				Description: "A sequence number representing a specific generation of the desired state. Populated by the system",
+				Type:        schema.TypeBigInt,
+				Resolver:    schema.PathResolver("ObjectMeta.Generation"),
+			},
+			{
+				Name:        "deletion_grace_period_seconds",
+				Description: "Number of seconds allowed for this object to gracefully terminate before it will be removed from the system",
+				Type:        schema.TypeBigInt,
+				Resolver:    schema.PathResolver("ObjectMeta.DeletionGracePeriodSeconds"),
+			},
+			{
+				Name:        "labels",
+				Description: "Map of string keys and values that can be used to organize and categorize (scope and select) objects",
+				Type:        schema.TypeJSON,
+				Resolver:    schema.PathResolver("ObjectMeta.Labels"),
+			},
+			{
+				Name:        "annotations",
+				Description: "Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata",
+				Type:        schema.TypeJSON,
+				Resolver:    schema.PathResolver("ObjectMeta.Annotations"),
+			},
+			{
+				Name:        "owner_references",
+				Description: "List of objects depended by this object",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveCoreEndpointsOwnerReferences,
+			},
+			{
+				Name:        "finalizers",
+				Description: "Must be empty before the object is deleted from the registry",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("ObjectMeta.Finalizers"),
+			},
+			{
+				Name:        "cluster_name",
+				Description: "The name of the cluster which the object belongs to. This is used to distinguish resources with same name and namespace in different clusters. This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request. +optional",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ObjectMeta.ClusterName"),
+			},
+			{
+				Name:        "managed_fields",
+				Description: "ManagedFields maps workflow-id and version to the set of fields that are managed by that workflow",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveCoreEndpointsManagedFields,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "k8s_core_endpoint_subsets",
```

Comment:
is it possible to avoid this relation inside relation?

Reply:
Didn't find a possible way as the children belong together and can't be easily grouped together otherwise.